src/themes/teams/components/Text/textStyles.ts

```diff
@@ -27,7 +27,15 @@ export default {
   }: TextStylesParams): ICSSInJSStyle => {
     return {
       ...(truncated && truncateStyle),
-      ...(atMention && { color: v.atMentionTextColor }),
+      ...(atMention &&
+        atMention === 'other' && {
```

Comment:
I would think that we would want to also handle the boolean case here as the "default" at mention formatting.  The exception to the `atMention` usage would be mentions to `me`.  So my proposal would be the majority case (just a boolean prop) would use the majority styling (other).

```jsx
atMention === true || atMention === 'other' && ...
```